pkg/kubectl/cmd/set/helper.go

```diff
@@ -21,20 +21,20 @@ import (
 	"io"
 	"strings"
 
+	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/strategicpatch"
-	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/legacyscheme"
```

Comment:
@liggitt this pkg should alse remove? we want remove all `kubernetes/kubernetes pkg` from kubectl?